packages/sit-onyx/src/components/OnyxMiniSearch/OnyxMiniSearch.vue

```diff
@@ -97,6 +103,7 @@ defineExpose({
       font-style: normal;
       flex-grow: 1;
       min-width: 0;
+      width: calc(var(--onyx-placeholder-character-count) * 1ch);
```

Comment:
Shouldn't it be min width so that the input width can be bigger than the placeholder?

Do we still need the `size` property with this approach?

Reply:
We need the size property because the native input component has a default property size with value 20, which is approximately 100px.